### ISSUE TITLE
ROS2 jazzy version

### ISSUE THREAD

**Issue by ZueVah** [Myzhar/ldrobot-lidar-ros2#25]
**Is your feature request related to a problem? Please describe.**
I am using a raspberry pi 5 so I can only use ros2 jazzy

**Describe the solution you'd like**
I request you share a ros2 jazzy package

**Comment by JustusBraun** [comment]
You can remove the ROS distribution check from the `CMakeLists.txt` file to compile for ROS2 Jazzy https://github.com/Myzhar/ldrobot-lidar-ros2/blob/9ea06d19ef5862d9335aa3e48442887551f2508e/ldlidar_component/CMakeLists.txt#L67-L76
I have done this for my use of this driver on a Pi4 and it works just fine :)

@Myzhar would it be possible to add `jazzy` to the list of supported ROS2 Distros? I can create a PR if desired :)

**Comment by Myzhar** [comment]
Yes, I want to add it in the next few weeks because I want to test new Nav2 features available only with Jazzy

**Comment by Myzhar** [comment]
Now the package supports Jazzy
